README.md

```diff
@@ -17,105 +17,97 @@ steps:
   with:
     fail_ci_if_error: true # optional (default = false)
     files: ./junit1.xml,./junit2.xml # optional
-    flags: py3.11 # optional
-    name: codecov-umbrella # optional
-    token: ${{ secrets.CODECOV_TOKEN }} # required
+    token: ${{ secrets.CODECOV_GITHUB_TOKEN }} # optional (default = false)
```

Comment:
by default I think we call this `CODECOV_TOKEN` everywhere

Reply:
in this case it's not actually a "codecov token" anymore, it needs to be a github personal access token, "codecov token" usually means then token you can get from the codecov website